dotCMS/src/integration-test/java/com/dotcms/contenttype/business/RelationshipAPITest.java

```diff
@@ -46,7 +55,9 @@
     private static Structure structure = null;
     private static User user = null;
     private static RelationshipAPI relationshipAPI = null;
+    private static ContentletAPI contentletAPI;
     private static ContentTypeAPI contentTypeAPI = null;
+    private static FieldAPI contentTypeFieldAPI = null;
```

Comment:
Codacy found an issue: [Avoid using redundant field initializer for 'contentTypeFieldAPI'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5775242)